lib/worker.js

```diff
@@ -10,8 +10,12 @@ const { Server } = metarhia.metacom;
 const { Auth } = require('./auth.js');
 
 (async () => {
+  const vm = require('vm');
+  const sandbox = { PORT: parseInt(process.env.PORT || 0) };
+  vm.createContext(sandbox);
+
   const configPath = path.join(application.path, 'config');
-  const options = { mode: process.env.MODE };
+  const options = { mode: process.env.MODE, sandbox };
```

Comment:
```suggestion
  const options = { mode: process.env.MODE, context };
```
We will rename sandbox to context in `metarhia/config` first, so `sandbox` will be used just in `impress` or other upper-level code but both `metavm` and `metarhia/config` will use `{ context }` in `options`